test/integration/local.js

```diff
@@ -15,13 +15,15 @@ const config = {
 function listAllEnvironmentVariables() {
   ;[
     'CONTENTFUL_SPACE_ID',
-    'CONTENTFUL_CMA_TOKEN',
     'CYPRESS_baseUrl',
     'TEST_LOCAL_SDK',
     'CONTENTFUL_LOCAL_TESTING_ENV'
   ].forEach(envvar => {
     console.log(`${envvar}=${process.env[envvar]}`)
   })
+  ;['CONTENTFUL_CMA_TOKEN'].forEach(envvar => {
+    console.log(`${envvar}=${process.env[envvar].slice(0, 5)}...`)
```

Comment:
Should this have `...` at the end? It isn't consistent with the other log on line 22.